pkg/controller/client/client_controller_config.go

```diff
@@ -63,18 +64,22 @@ func (c ClientControllerConfig) ApplyConfigMap(configMap *corev1.ConfigMap, keys
 	if configMap.Data != nil || len(configMap.Data) == 0 {
 		if len(keysList) != 0 {
 			for _, k := range keysList {
-				if k == authServiceURLKey || k == identityProviderURLKey || k == identityManagementURLKey {
+				if (k == authServiceURLKey || k == identityProviderURLKey || k == identityManagementURLKey) && !strings.Contains(configMap.Data[k], "127.0.0.1") {
```

Comment:
I don't think I agree with this approach - in the event that a change happens to a ConfigMap/Secret that this code caches data from, that cache isn't updated. We'd still need to restart the Operator Pod to make this work, at which point we may as well just do that instead. Am I missing something?